Dockerfile

```diff
@@ -32,18 +32,23 @@ RUN adduser --disabled-login --disabled-password ubuntu --ingroup ubuntu
 # Give Ownership of sandbox env to sandbox group and user
 RUN chown -R $CODEJAIL_TEST_USER:$CODEJAIL_GROUP $CODEJAIL_TEST_VENV
 
-# Clone Codejail Repo
-ADD . ./codejail
 WORKDIR /codejail
 
+# Clone Requirement files
+COPY ./requirements/sandbox.txt ./codejail/requirements/sandbox.txt
+COPY ./requirements/testing.txt ./codejail/requirements/testing.txt
+
 # Install codejail_sandbox sandbox dependencies
 RUN source $CODEJAIL_TEST_VENV/bin/activate && pip install -r requirements/sandbox.txt && deactivate
 
 # Install testing requirements in parent venv
 RUN pip install -r requirements/sandbox.txt && pip install -r requirements/testing.txt
 
+# Clone Codejail Repo
+COPY . ./codejail
+
 # Setup sudoers file
-ADD sudoers-file/01-sandbox /etc/sudoers.d/01-sandbox
+COPY sudoers-file/01-sandbox /etc/sudoers.d/01-sandbox
```

Comment:
We are just copying local files and hence COPY command is the most suitable one for that.